15-node-js-express/backend/index.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+const cafecito_express = require('express');
```

Comment:
```suggestion
const express = require('express');
```
use a better const name. I know it's fun but let's keep it professional all the time.

Reply:
Ok i forgot to change that, jeje was for better learning